AnnoyancesFilter/sections/cookies_general.txt

```diff
@@ -1,3 +1,10 @@
+!
+! This section contains the list of the rules are against cookies. Rules should be generic.
```

Comment:
I guess 'cookie notifications' would be more appropriate. Same for the `cookies specific` file.

Reply:
Yep, I can change it.